gqcp/include/ONVBasis/SpinUnresolvedOperatorString.hpp

```diff
@@ -71,40 +73,72 @@ class SpinUnresolvedOperatorString {
         p {phase_factor} {};
 
 
+    /*
+     * MARK: Named constructors
+     */
+
+    /**
+     *  Construct a `SpinUnresolvedOperatorString` from the occupied indices of a `SpinUnresolvedONV`.
+     * 
+     *  @param onv                The `SpinUnresolvedONV` encapsulating the operator indices of the ONV.
+     */
+    static SpinUnresolvedOperatorString FromONV(const SpinUnresolvedONV& onv) { return SpinUnresolvedOperatorString(onv.occupiedIndices()); };
+
+
     /*
      *  MARK: General information
      */
 
     /**
      *  Retrieve the operator indices from the `SpinUnresolvedOperatorString`.
+     * 
+     *  @return The vector containing the operator indices.
      */
     const std::vector<size_t>& operatorIndices() const { return this->indices; }
 
     /**
      *  Retrieve the phase factor corresponding to the `SpinUnresolvedOperatorString`.
+     * 
+     *  @return The phase factor associated with the operator string.
      */
     int phaseFactor() const { return this->p; }
 
     /**
      *  Check whether the operator string in question will result in zero when applied to the wave function.
      *
      *  Note: There are different cases when an operator string will result in a zero value. This method checks all of them.
```

Comment:
```suggestion
```

This was the case initially, but after some thought only one condition needed to be checked. So this note in the documentation can be removed.